main.js

```diff
@@ -84,8 +83,7 @@ function enableSaveButton() {
 
 function saveOutfit() {
   var curId = parseInt(localStorage.getItem('id'));
-  currentOutfit.id = curId;
-  localStorage.setItem('id', ++curId);
+  currentOutfit.id = new Date().valueOf();
```

Comment:
since we are using this id as an html id as well, it needs to start with a letter

Reply:
Is that in the style guide or something?

Reply:
Done